bin/git-forgit

```diff
@@ -137,10 +137,10 @@ _forgit_diff() {
     #   oldfile\0
     # We have to do a two-step sed -> tr pipe because OSX's sed implementation does
     # not support the null-character directly.
-    get_files="echo {} | sed 's/.*] *//' | sed 's/  ->  /\\\n/' | tr '\\\n' '\\\0'"
+    get_files="echo {} | sed 's/\\\\s*\\\\[.]\\\\s*//' | sed 's/  ->  /\\\n/' | tr '\\\n' '\\\0'"
```

Comment:
Oh wow, respect that you fixed this on the current codebase instead of the new version without the deferred code. This looks so ugly, haha! I am so looking forward to get rid of all these escapes. 😅

Reply:
Yeah, I actually did the fix in the new codebase and backported it. Troubleshooting issues like this is **a lot** easier in the new code base IMO :slightly_smiling_face: